Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            
+        }
+        public static int PromptInt(string message)
+            {
+                Console.Write(message);
+                int intedMessage = int.Parse(Console.ReadLine());
```

Comment:
You're not getting a stringed message- you're getting a number.

Reply:
I can't get my head around it. Aren't i getting a message (in string) for requesting to enter a number and then parsing the input into int?

Reply:
Console.ReadLine()- this is getting the user input
int.Parse() - this is turned that to an int
The result?
You have a number.

Reply:
![image](https://user-images.githubusercontent.com/66484094/88097661-2fc78700-cba1-11ea-8a59-7fbd55a2abd0.png)

Reply:
> Console.ReadLine()- this is getting the user input
> int.Parse() - this is turned that to an int
> The result?
> You have a number.
My code does exactly that. 
Pormpting int(age) asks for age, int.parse parses the readlined number and it does give number.